src/AppBundle/Entity/User.php

```diff
@@ -78,9 +78,9 @@ class User extends BaseUser
     /**
      * @var bool
      *
-     * @ORM\Column(type="boolean")
+     * @ORM\Column(type="boolean", nullable=true)
```

Comment:
Remove the default value like `protected $wantToLearn;`. The `ORM`-annotation is for database schema.